src/cli/post-install.ts

```diff
@@ -30,7 +30,7 @@ import { ApplicationConfigFolder, LocalConfigFile } from "$common/config/ConfigF
 
     // We should instead move this to UI
     // configObject.sendTelemetryData = await cliConfirmation(
-    //     "We collect usage info to improve the data modeler. " +
+    //     "We collect usage info to improve Rill Developer. " +
```

Comment:
Just noticed this - how is this surfaced? I want to make sure we are collecting analytics by default and it looks like it is encouraging users to opt out.

Reply:
It's not surfaced, just a comment right now. Should we delete it?